app/ipc-types.ts

```diff
@@ -0,0 +1,368 @@
+/* eslint-disable camelcase */
+/* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable quotes */
+
+export enum WorkerStatus {
+  runningInBackground = `RUNNING_IN_BACKGROUND`,
+  stopped = `STOPPED`,
+}
+
+export type Status = GlobalStatus | WorkerStatus
+
+/**
+ * Structured logs types. This is taken from Cloud.
+ */
+
+export enum StructuredLogLevel {
+  Log = `LOG`,
+  Warning = `WARNING`,
+  Info = `INFO`,
+  Success = `SUCCESS`,
+  Error = `ERROR`,
+  Debug = `DEBUG`,
+}
+
+export enum JobsApiNames {
+  ImageProcessing = `IMAGE_PROCESSING`,
+}
+
+enum ActivityLogLevel {
+  ActivitySuccess = `ACTIVITY_SUCCESS`,
+  ActivityFailed = `ACTIVITY_FAILED`,
+  ActivityInterrupted = `ACTIVITY_INTERRUPTED`,
+}
```

Comment:
This file feels like an indication to me that we might benefit from a `gatsby-types` package like jest has.

Reply:
Yes, that's something I've thought a few times.